src/huggingface_hub/repository.py

```diff
@@ -317,8 +317,9 @@ def _lfs_log_progress():
     if logger.getEffectiveLevel() >= logging.ERROR:
         try:
             yield
-        finally:
-            return
+        except Exception:
+            pass
+        return
 
```

Comment:
@LysandreJik I want to double-check that with you. Is this the intended purpose to silently catch any exception and return in all cases ? I saw that it has been introduced by https://github.com/huggingface/huggingface_hub/pull/297.

`flake8-bugbear` advice against the try/finally/return pattern because exceptions are silenced without being explicit for the developer. It's only a readability concern.

Reply:
The intended purpose was for this context manager to act as a no-op in case the logger has a level superior or equal to `ERROR`. That seems to be satisfied by your approach, so fine by me to have this in!